cql2pgjson/src/test/java/org/folio/cql2pgjson/CQL2PgJSONTest.java

```diff
@@ -902,22 +904,42 @@ public void allAnyAdjFT(String testcase)
-    "cql.allRecords=1 NOT lang <>/respectAccents []     # a; n",
-    "lang =/respectCase/respectAccents en               # b; c; d; f; g; h; i",
-
-    // note that \"en\" also matches case f ["\"en"]
-    "                     lang =/respectCase/respectAccents \\\"en\\\"   # b; f; i",  // without Java quoting: \"en\"
-    "cql.allRecords=1 NOT lang =/respectCase/respectAccents \\\"en\\\"   # a; c; d; e; g; h; n",
-    "lang = \"\"      NOT lang =/respectCase/respectAccents \\\"en\\\"   # a; c; d; e; g; h",
-    "lang = \"\"                                                         # a; b; c; d; e; f; g; h; i",
-    "cql.allRecords=1 NOT lang = \"\"                                    # n",
-   */})
+    "lang == \"en\"                #",
+    "lang = en                     # b; c; d; f; g; h; i",
+    "lang = au dk                  #",
+    "lang all au dk                # i",
+    "lang = de dk                  # i",
+    "lang = \"en-uk\"              # d",
+    "lang = en-uk                  # d",
+    "lang = uk-en                  #",
+    "lang = uk                     # d; e",
+    "contributors = terry          # a",
+    "contributors = \"2b94c631-fca9-4892-a730-03ee529ffe2a\" # a",
+    "contributors =/@contributornametypeid=2b94c631-fca9-4892-a730-03ee529ffe2a terry # a",
+    "contributors =/@contributornametypeid=2b94c631-fca9-4892-a730-03ee529ffe2A terry # ", // notice that it's case sensitive
+    "contributors =/@contributornametypeid=2b94c631-03ee529ffe2a terry # ",
+    "contributors =/@contributornametypeid=\"2b94c631-fca9-4892-a730-03ee529ffe2a\" creator #",
+    "contributors =/@contributornametypeid=e8b311a6-3b21-43f2-a269-dd9310cb2d0a creator a #",
+    "contributors =/@contributornametypeid=e8b311a6-3b21-43f2-a269-dd9310cb2d0a creator b # b",
+    "contributors =/@contributornametypeid=e8b311a6-3b21-43f2-a269-dd9310cb2d0a creator c #",
+    "contributors =/@foo=e8b311a6-3b21-43f2-a269-dd9310cb2d0a creator c #CQL: Unsupported relation modifier @foo ",
+    "contributors =/@foo=1/@bar=2 c #CQL: Only one @-relation modifier supported ",
+    "contributors =/@foo<1 c #CQL: Unsupported comparison for relation modifier @foo ",
+    "identifiers =/@contributornametypeid=e8b311a6-3b21-43f2-a269-dd9310cb2d0a c #CQL: Unsupported relation modifier @contributornametypeid",
+    "identifiers =/@contributornametypeid c #CQL: Missing value for relation modifier @contributornametypeid",
+    "contributors all \"contributornametypeid\" # a; b",
+    "contributors all \"contributorNametypeid 2b94c631-fca9-4892-a730-03ee529ffe2a\" # a",
+    "contributors all \"contributorNametypeid 2b94c631\" # a",
+    "contributors all \"contributorNametypeid 4892\" # a",
+    "contributors = \"contributorNameTypeId 2b94c631\" #",
+    "identifiers = \"value 0552142352\" # a",
+    "contributors = name           # a; b",
+    "name = terry                  #",
+    "contributors.name = terry     #",
+    "contributors.name = terry     #",
+    })
```

Comment:
Can you add test cases for 1 level and 2 level nested array like personal.addresses and contactInformation.phone.mobile?

Reply:
Can you add test cases for exact match comparison using ==, for example

    "contributors ==/@contributornametypeid==e8b311a6-3b21-43f2-a269-dd9310cb2d0a creator "B. Maya"

Reply:
Sure.. Something added in this.